app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -144,10 +153,37 @@ export default function ResourceAuthoringPage() {
     name.trim() !== '' ? (additions['name'] = name) : (deletions['name'] = '');
     title.trim() !== '' ? (additions['title'] = title) : (deletions['title'] = '');
     description.trim() !== '' ? (additions['description'] = description) : (deletions['description'] = '');
+    library ? (additions['library'] = [library]) : (deletions['library'] = null);
 
     return [additions, deletions];
   }
 
+  // set up main library options
+  let libOptions: { value: string; label: string; disabled: boolean }[] = []; // default to empty
+  if (resourceType === 'Measure') {
+    const { data: libraries } = trpc.draft.getDrafts.useQuery('Library' as ArtifactResourceType);
+    if (libraries) {
+      libOptions = libraries?.map(l => {
+        if (l.url) {
+          // prioritizes use of url/version
```

Comment:
Do we need to include `version` stuff here at all? Since we are only looking at draft libraries I don't think there will ever be versions.

Reply:
I kind of left it in in case we were ever dealing with non-draft libraries or reusing it for anything that has version. This is technically how the canonical reference works: we just happen to know that there won't be a version in the current context. But can definitely remove if we don't think it's ever going to be useful!

Reply:
Gotcha! Doesn't hurt to leave in